deployment.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: biovalidator
+  name: biovalidator
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: biovalidator
+  template:
+    metadata:
+      labels:
+        app: biovalidator
+    spec:
+      imagePullSecrets:
+        - name: $DOCKER_PULL_SECRET
+      containers:
+        - name: biovalidator
```

Comment:
Add a [liveness prob](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/)e so k8s can know whether the app is running.